test/node-next/es2018/cursor_async_iterator.test.js

```diff
@@ -83,7 +83,7 @@ describe('Cursor Async Iterator Tests', function() {
           expect(doc).to.exist;
           cursor.close();
         }
-        throw new Error('expected closing the cursor to break iteration');
+        throw new MongoError('expected closing the cursor to break iteration');
```

Comment:
this change looks wrong to me. I think the test is trying to signal that something went wrong by throwing the `Error` here, otherwise the `catch` below will swallow it.